### DISCUSSION TITLE
YAML config for RGB lights, color temp, RGB, and lighting modes.

### DISCUSSION THREAD

**Discussion by bobloadmire** [Clooos/Bubble-Card#772] [Questions about config, custom styles and templates]
Has anyone successfully made a Bubble Card for RGB color strips? Want to be able to slide for color temp, as well as select modes like “sunset, audio sync” etc. Looked through the info, while there are a lot of great examples, I didn't see anything that I could modify for this sort of usage. any suggestions?

**Comment by MrBearPresident** [comment]
Have you solved this?
Can this be closed?
